models/Post.php

```diff
@@ -46,7 +46,7 @@ class Post extends Model
     ];
 
     /**
-     * @var array Attributes to be stored as JSON
+     * @var array Attributes to be stored as JSON.
```

Comment:
Please don't add periods to the end of docblocks, and also please stop removing trailing commas from PHP arrays. I prefer to leave the trailing commas in the array so that if the last item is copy-pasted to make a new one without updating commas the array is still valid.

Reply:
Alright.